include/z64.h

```diff
@@ -892,6 +892,8 @@ typedef struct {
     /* 0x74 */ s16 unk_74[2]; // context-specific data used by the current scene draw config
 } RoomContext; // size = 0x78
 
+#define SAC_ENABLE (1 << 0)
```

Comment:
right below this line is `CollisionCheckContext`

Reply:
I guess I'm more asking why are we moving defines out of the headers they belong in instead of moving the structs out of the massive header blob into the headers they belong in

Reply:
I agree `CollisionCheckContext` should move to the adequate header
I didn't target header cleanup with this PR though, more the collider system itself